frontend_addon/cookiecutter.json

```
@@ -6,6 +6,7 @@
   "email": "collective@plone.org",
   "github_organization": "collective",
   "npm_package_name": "{{ cookiecutter.frontend_addon_name }}",
+  "year": "{% now 'utc', '%Y' %}",
```

For the sad lonely developer working on one side of the UTC timezone or the other on New Year's Eve.

```suggestion
  "year": "{% now 'local', '%Y' %}",
```

See https://github.com/hackebrot/jinja2-time?tab=readme-ov-file#usage

---

Done